app/models/administrator.rb

```diff
@@ -1,4 +1,12 @@
 class Administrator < ApplicationRecord
+  def self.authenticate(username, password)
+    admin = Administrator.find_by(username: username)
+    if admin.nil?
+      false
+    else
+      admin.password == password
```

Comment:
This can be replaced by modifying the find query:

`find_by(username: username, password: password)`